server/utils/nunjucksSetup.test.ts

```diff
@@ -1,190 +1,90 @@
-import * as cheerio from 'cheerio'
-import nunjucks, { Template } from 'nunjucks'
-import { registerNunjucks } from './nunjucksSetup'
+import express from 'express'
+import { FieldValidationError } from 'express-validator'
+import nunjucksSetup from './nunjucksSetup'
 
 describe('Nunjucks Filters', () => {
-  let compiledTemplate: Template
-  let viewContext: Record<string, unknown>
-
-  const njkEnv = registerNunjucks(null)
-
-  describe('initialiseName', () => {
```

Comment:
Removed tests of a few of the filters where they had duplicate coverage in `utils.test.ts` because  they just call functions in `utils.ts.